src/BundleAdjustmentNLSFunctions.jl

```diff
@@ -1,7 +1,13 @@
-# Manual jacobian functions are not proven
-
 import NLPModels: increment!
 
+export BundleAdjustmentModel,
+  residual!,
+  residuals!,
+  jac_structure!,
+  jac_coord!,
+  jac_op_residual,
```

Comment:
The user is supposed to do `using NLPModels` to access those.

Reply:
Sorry, when I updated some code in an other package I didn't realise I didn't type the "using NLPModels"
I will change that